sources/packages/web/src/components/common/students/StudentApplications.vue

```diff
@@ -41,7 +41,11 @@
               >
             </v-btn>
 
-            <span v-if="clientType === ClientIdType.AEST"
+            <span
+              v-if="
+                clientType === ClientIdType.AEST ||
```

Comment:
Since it is an old code (and refactoring would not be part of the PR) I will not request to change it but doing the checks based on the client is what we have been trying to avoid.

Reply:
IMO we can do a v-if based on optional prop value which it's consumer can use.

Reply:
I have added props to take care of these actions